core/shared/src/test/scala/eu/timepit/refined/BooleanValidateSpec.scala

```diff
@@ -4,18 +4,18 @@ import eu.timepit.refined.TestUtils._
 import eu.timepit.refined.api.Validate
 import eu.timepit.refined.boolean._
 import eu.timepit.refined.char._
-import eu.timepit.refined.numeric.{ Greater, Less }
+import eu.timepit.refined.numeric.{Greater, Less}
 import org.scalacheck.Prop._
 import org.scalacheck.Properties
 import shapeless.nat._
-import shapeless.{ ::, HNil }
+import shapeless.{::, HNil}
 
 class BooleanValidateSpec extends Properties("BooleanValidate") {
 
-  type FF[Op[_, _]] = False Op False
-  type FT[Op[_, _]] = False Op True
-  type TF[Op[_, _]] = True Op False
-  type TT[Op[_, _]] = True Op True
+  type FF [Op[_, _]] = False Op False
+  type FT [Op[_, _]] = False Op True
+  type TF [Op[_, _]] = True Op False
+  type TT [Op[_, _]] = True Op True
```

Comment:
There should be a space, this is fixed in the already released 0.2.2 but because that release introduced https://github.com/olafurpg/scalafmt/issues/192 I haven't advertised it.